cs.html

```diff
@@ -961,689 +1090,685 @@ <h2>Constructors</h2>
-             boolean finished = false;
-        };
-        </pre>
-        <section>
-          <h2>Dictionary <a class="idlType">QuicStreamReadResult</a> Members</h2>
-          <dl data-link-for="QuicStreamReadResult" data-dfn-for="QuicStreamReadResult" class=
-          "dictionary-members">
-            <dt><dfn><code>amount</code></dfn> of type <span class=
-            "idlMemberType"><a>unsigned long</a></span>.</dt>
-            <dd>
-              <p>The amount of data read in bytes.</p>
-            </dd>
-            <dt><dfn><code>finished</code></dfn> of type <span class=
-            "idlMemberType">boolean</span>.</dt>
-            <dd>
-              <p>Set to <code>true</code> if the <code><a>QuicReadableStream</a></code> has
-              <a>finished reading</a>.</p>
-            </dd>
-          </dl>
-        </section>
-      </div>
-    </section>
-    <section id="quicstreamabortinfo*">
-      <h3><dfn>QuicStreamAbortInfo</dfn> Dictionary</h3>
-      <p>The <code>QuicStreamAbortInfo</code> dictionary includes information
-      relating to the error code for aborting a QUIC stream. This could be used either
-      in a RST_STREAM frame or STOP_SENDING frame.</p>
-      <div>
-        <pre class="idl">dictionary QuicStreamAbortInfo {
-             unsigned short errorCode = 0;
-        };
-        </pre>
-        <section>
-          <h2>Dictionary <a class="idlType">QuicStreamAbortInfo</a> Members</h2>
-          <dl data-link-for="QuicStreamAbortInfo" data-dfn-for="QuicStreamAbortInfo" class=
-          "dictionary-members">
-            <dt><dfn><code>errorCode</code></dfn> of type <span class=
-            "idlMemberType"><a>unsigned short</a></span>.</dt>
-            <dd>
-              <p>The error code used in the RST_STREAM or STOP_SENDING frame.
-              The default value of 0 means "STOPPING."</p>
-            </dd>
-          </dl>
-        </section>
-      </div>
-    </section>
+      <section id="quicstreamresult*">
+        <h3><dfn>StreamReadResult</dfn> Dictionary</h3>
+        <p>The <code>StreamReadResult</code> dictionary includes information
+        relating to the result returned from <code>readInto</code>.</p>
+        <div>
+          <pre class="idl">dictionary StreamReadResult {
+                unsigned long amount;
+                boolean finished = false;
+          };
+          </pre>
+          <section>
+            <h2>Dictionary <a class="idlType">StreamReadResult</a> Members</h2>
+            <dl data-link-for="StreamReadResult" data-dfn-for="StreamReadResult" class=
+            "dictionary-members">
+              <dt><dfn><code>amount</code></dfn> of type <span class=
+              "idlMemberType"><a>unsigned long</a></span>.</dt>
+              <dd>
+                <p>The amount of data read in bytes.</p>
+              </dd>
+              <dt><dfn><code>finished</code></dfn> of type <span class=
+              "idlMemberType">boolean</span>.</dt>
+ 
```

Comment:
Could change to QuicReceiveStream and QuicBidirectionalStream to avoid the name conflict?

Reply:
I was trying to have the stream names apply to non-QUIC transports as well.